packages/web/pages/history.tsx

```diff
@@ -0,0 +1,40 @@
+import { CombatStubList } from '@wowarenalogs/shared';
+import { LocalRemoteHybridCombat } from '@wowarenalogs/shared/src/components/CombatStubList/rows';
+import { QuerryError } from '@wowarenalogs/shared/src/components/common/QueryError';
+import { useGetMyMatchesQuery } from '@wowarenalogs/shared/src/graphql/__generated__/graphql';
+import _ from 'lodash';
+import { useMemo } from 'react';
+import { TbLoader } from 'react-icons/tb';
+
+const Page = () => {
+  const matchesQuery = useGetMyMatchesQuery();
+
+  const hybridCombats = useMemo(() => {
+    const remoteCombats = matchesQuery.data?.myMatches?.combats || [];
+    return _.orderBy(
+      remoteCombats.map((c) => ({
+        isLocal: false,
+        isShuffle: c.__typename === 'ShuffleRoundStub',
+        match: c,
+      })) as LocalRemoteHybridCombat[],
+      (c) => c.match.startTime,
+      ['desc'],
+    );
+  }, [matchesQuery.data]);
+
+  return (
+    <div className="transition-all px-2 overflow-y-auto">
+      <div className="animate-fadein mt-2">
+        <CombatStubList viewerIsOwner={true} combats={hybridCombats} source="history" />
```

Comment:
Maybe missing an "Empty" display here -- anon user with no matches uploaded just shows a blank page at /history

Could also show a prompt to log in if they aren't